10.0/base_requirements.txt

```diff
@@ -63,7 +63,7 @@ urllib3==1.24.2
 
 # Migration tools
 ruamel.yaml==0.16.13
-marabunta==0.10.5
+marabunta==0.10.6
```

Comment:
Not available as we need to update the way we are creating new verions of marabunta